5-logging/bookshelf/__init__.py

```diff
@@ -70,15 +71,17 @@ def logout():
     def index():
         return redirect(url_for('crud.list'))
 
-    # Add an error handler. This is useful for debugging the live application,
-    # however, you should disable the output of the exception for production
-    # applications.
+    # Add an error handler that reports exception to Stackdriver Error
+    # Reporting. Note that this Error Handler is only used when Debug
```

Comment:
No need to capitalize Error Handler or Debug

![I am error](https://professionalmorondotcom.files.wordpress.com/2015/03/zelda-ii-i-am-error.png?w=742&h=345)